src/webextension/firstrun/index.js

```diff
@@ -0,0 +1,13 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from "react";
+import ReactDOM from "react-dom";
+
+import Wizard from "./wizard";
+
+ReactDOM.render(
+  <Wizard/>,
```

Comment:
Nit: I don't have a strong preference, but I generally see people call the root component `<App/>`.